distrolesss.dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM docker.io/library/golang:1.17.8 as builder
 
 # Install FDB this version is only required to compile the fdb operator
-ARG FDB_VERSION=6.3.23
+ARG FDB_VERSION=6.3.24
```

Comment:
TODO revert back to 6.3.24, see: https://github.com/apple/foundationdb/issues/6872